infrastructure/modules/monitoring/fluent-bit/configs/input/input-kubernetes-sidecar.conf

```diff
@@ -18,7 +18,22 @@
     Tag                application.*
     Path               /var/log/containers/${HOSTNAME}*.log
     Exclude_Path       /var/log/containers/${HOSTNAME}*${FLUENT_CONTAINER_NAME}*.log
-    Parser             docker
+    Parser             cri
```

Comment:
Is this a safe change?

Reply:
with docker we can't see correctly the logs with the aws deployement. I think we have to variabilise this . (docker -> localhost , cri -> aws)